docs/moduleAna3AIMEC.rst

```diff
@@ -23,20 +26,24 @@ Inputs
 *  avalanche path in LINES (as a shapefile named ``path_aimec.shp``)
 *  a splitPoint in POINTS (as a shapefile named ``splitPoint.shp``)
 *  Results from avalanche simulation (when using results from com1DFA,
-   the helper function :py:func:`ana3AIMEC.dfa2Aimec.mainDfa2Aimec` in :py:mod:`ana3AIMEC.dfa2Aimec` fetches and prepares the input for Aimec)
+   the helper function :py:func:`ana3AIMEC.dfa2Aimec.mainDfa2Aimec` in
+   :py:mod:`ana3AIMEC.dfa2Aimec` fetches and prepares the input for Aimec)
+* a method to define the reference simulation. By default, an arbitrary simulation
+  is defined as reference (this can be changed in the ``ana3AIMEC/local_ana3AIMECCfg.ini``
+  as explained in :ref:`moduleAna3AIMEC:Defining the reference simulation`)
```

Comment:
```suggestion
  is defined as reference. This can be changed in the ``ana3AIMEC/local_ana3AIMECCfg.ini``
  as explained in :ref:`moduleAna3AIMEC:Defining the reference simulation`.
```